store/tikv/lock_resolver.go

```diff
@@ -229,9 +305,11 @@ func (lr *LockResolver) getTxnStatus(bo *Backoffer, txnID uint64, primary []byte
 			return status, errors.Trace(err)
 		}
 		resp, err := lr.store.SendReq(bo, req, loc.Region, readTimeoutShort)
+		tempTime := time.Now()
 		if err != nil {
 			return status, errors.Trace(err)
 		}
+		lr.costTxnStatus += time.Since(tempTime)
```

Comment:
What is the purpose here?

Reply:
```
	log.Infof("BatchResolveLocks: it takes %v cost to lookup %v txn status from %v txn status", lr.costTxnStatus, lr.cntTxnStatus, count)
```
used for log.

Reply:
It will be near to 0. `tempTime := time.Now()` should go before `SendReq`.

Reply:
yes, I will fix this.